src/zero-md.js

```diff
@@ -166,8 +166,12 @@ class ZeroMd extends HTMLElement {
     let path = ev.path || ev.composedPath();
     if (path[0].tagName !== 'A') { return; }
     ev.preventDefault();
-    this._scrollTo(path[0].hash);
-    window.location = path[0].href;
+    if (ev.metaKey) {
+      window.open(window.location.href  + path[0].hash, '_blank');
```

Comment:
Bug here. This is opening the current page in the new tab, not the actual link.

How about let the browser do its thing:

```js
_hijackLinks(ev) {
  let path = ev.path || ev.composedPath();
  if (path[0].tagName !== 'A') { return; }

  const link = path[0];
  if (link.hash && link.href === window.location.href)
    this._scrollTo(path[0].hash);
    ev.preventDefault();
  }
}
```

Reply:
Only `preventDefault` in the special use case of anchor scroll.

Reply:
Good idea, I think that will work. Do you mind opening a PR?

Reply:
Sure, in a bit.

Reply:
Opened https://github.com/zerodevx/zero-md/pull/10